cmd/helpers.go

```diff
@@ -111,39 +236,21 @@ func BuildApplication(binaryName string, forceRebuild bool, buildMode string, pa
 				binaryName = strings.TrimSuffix(binaryName, ".exe")
 			}
 		}
-		buildCommand.Add("-o", binaryName)
+		buildCommand.Add("-o", filepath.Join("build", binaryName))
```

Comment:
This breaks my `wails serve` command as well.

To fix it, please change line 536 of this file with:

```go
location, err := filepath.Abs(filepath.Join("build", projectOptions.BinaryName))
```